common/models/report-model.js

```diff
@@ -1226,9 +1226,13 @@ module.exports = function (ReportModel) {
                     apiVersion: '2006-03-01',
                     region: ReportModel.app.get('awsS3Region'),
                     accessKeyId: ReportModel.app.get('awsAccessKeyId'),
-                    secretAccessKey: ReportModel.app.get('awsSecretAccessKey')
+                    secretAccessKey: ReportModel.app.get('awsSecretAccessKey'),
+                    endpoint: process.env.ENDPOINT,
```

Comment:
`ENDPOINT` is very vague 
change it to `S3_ENDPOINT` or some thing that makes more sense 

Also this env variable is missing in .env.example file

Reply:
Done